apps/aecore/lib/aecore/account/tx/spend_tx.ex

```diff
@@ -114,14 +114,16 @@ defmodule Aecore.Account.Tx.SpendTx do
           tx_type_state(),
           non_neg_integer(),
           SpendTx.t(),
-          DataTx.t()
+          DataTx.t(),
+          Transaction.context()
```

Comment:
what is context in general?

Reply:
for now, it is only used in the contract call tx; that is to know if the call is from the top level, in order to charge the caller fee and gas

Reply:
So, could we give it a contract specific name, with including it in the callback functions it looks like context important for all transactions